src/features/game/expansion/components/ClaimReward.tsx

```
@@ -205,72 +212,110 @@ export const Rewards: React.FC<{
+                        className="ml-1 mb-1"
+                      >{`+${formatNumber(
+                        getFoodExpBoost({
+                          food: CONSUMABLES[name as ConsumableName],
+                          game,
+                          createdAt: now,
+                        }).boostedExp,
+                        { decimalPlaces: 0 },
+                      )} XP`}</Label>
+                    )}
+                    {rewardBoxName && (
+                      <Label type="default" className="ml-auto"
```

ESLint

[error] Replace `·type="default"` with `⏎························type="default"⏎·······················`

(prettier/prettier)